Oasis Welness  W-12

Maria-Manuela opened the pull request:
## Netlify link
https://oasiswelnesscenter.netlify.app/

## Collaborators

[github-Looziolooz]

AntonellaMorittu commented on src/components/GlobalStyle.jsx:
please format this file 🧹

AntonellaMorittu reviewed (approved):
Hello Maria-Manuela and Lorenzo, big congrats for completing your design!  It looks ✨ awesome ✨ The site looks very close to the design and followed our requirements. How did you like Styled Components? Seems like you tackled it in the right way and implemented a great result, the button styled component is a great example! Now the design is not pixel perfect, but that's okay, here's a little things that can be taken into consideration to improve it even further:

- footer: alignment of buttons should be centered (justify-content: space-around)
- on tablet and mobile: card corner doesn’t match the design

A requirement was to make the app responsive on screens starting from 320px:
- on mobile: 95 - Keep an eye on responsiveness on small screen

Fix these little things to pass the assignment, really well done 👏

Maria-Manuela commented:
I really loved to worked with styled components, for someone like me who loose the focus so easily, this gave more control over the styling. It´s a bless!
I fixed the issues and adjusted more things that were not according to the desgin. Now should look better also on the mobile @AntonellaMorittu
